forms/Form.php

```diff
@@ -152,17 +184,25 @@ class Form extends RequestHandler {
 	 */
 	protected $attributes = array();
 
+	/**
+	 * @var array
+	 */
 	private static $allowed_actions = array(
-		'handleField', 
+		'handleField',
 		'httpSubmission',
 		'forTemplate',
 	);
 
 	/**
+	 * @var bool
+	 */
+	protected $securityTokenAdded;
```

Comment:
It seems this var just isn't declared anywhere, is that right? so this is effectively declaring it's visibility.

If it's `@var bool` then you either need to assign it an initial value, or you need to do `@var bool|null`